### PR TITLE
This should fix an intermittent problem with ICEFinder

### PR DESCRIPTION
**Pull request by mberacochea**
The problem was that ICEFinder was mounting files from the params.outdir, those files were from publishDir. So, it happened from time to time that the files were not there to be mounted.

### CODE REVIEW

File: modules/icefinder.nf [hunk 1]
```diff
@@ -18,24 +18,24 @@ process ICEFINDER {
...
         args = [
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/input.list:/install/ICEfinder_linux/input.list",
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/gbk/:/install/ICEfinder_linux/gbk/",
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/tmp/:/install/ICEfinder_linux/tmp/",
-            "--bind $PWD/$params.outdir/prediction/icefinder_results/result/:/install/ICEfinder_linux/result/",
+            "--bind ${input_list}:/install/ICEfinder_linux/input.list",
+            "--bind ${gbk_folder}:/install/ICEfinder_linux/gbk/",
+            "--bind ${tmp_folder}:/install/ICEfinder_linux/tmp/",
+            "--bind ${res_folder}:/install/ICEfinder_linux/result/",
             "--pwd /install/ICEfinder_linux/"
```
Comment (chrisAta) [inline modules/icefinder.nf:25]: Would it be worth making this usable with Docker too? like this (from the interproscan nf-module):
```
    containerOptions {
        if (workflow.containerEngine == 'singularity') {
            return "--bind ${interproscan_db}/data:/opt/interproscan-5.62-94.0/data"
        } else {
            return "-v ${interproscan_db}/data:/opt/interproscan-5.62-94.0/data"
        }
    }
```

File: modules/icefinder.nf [hunk 2]
```diff
@@ -48,14 +48,14 @@ process ICEFINDER {
...
         fi
         """
     else
         """
         echo 'No input files for ICEfinder... generating dummy files'
-	    touch result/icf_concat.summary
+        touch result/icf_concat.summary
         touch result/icf_dr.txt
         """
 }
```
Comment (chrisAta) [inline modules/icefinder.nf:61]: missing newline at end of file

**Review by chrisAta — APPROVED** [review]
Looks good to me! Two small comments

File: modules/icefinder.nf (hunk 1 above)
Comment (mberacochea) [reply modules/icefinder.nf:25]: We have to do that. But, ATM the pipeline lacks a bit as that container needs to be built by the user (licensing issues). We will re-work that bit in due time

File: modules/icefinder.nf (hunk 2 above)
Comment (mberacochea) [reply modules/icefinder.nf:61]:
```suggestion
}

```

**Pull request merged**